SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+The Fuel Typescript SDK is still in development, which means that our implementation may have issues.
+
+We take security vulnerabilities very seriously. If you discover a security issue, please immediately bring it to our attention!
+
+## Reporting a Vulnerability
+
+If you find a vulnerability or exploit, please [**report privately**](https://github.com/FuelLabs/fuels-ts/security/advisories/new).
+
+Please **DO NOT file a public issue**.
+
+We will send a response indicating the next steps to handle your report. The security team will let you know about the progress toward a fix and full announcement and may ask for additional information or guidance.
+
+If the issue is an implementation weakness that cannot be immediately exploited or something that has not yet been deployed, just discuss it openly.
+
+## Reporting a non security bug
+
+For non-security bugs, please simply file a GitHub [issue](https://github.com/FuelLabs/fuels-ts/issues/new).
```

Comment:
I think we should be more direct, neutral, and broad in this communication.

<img width="570" alt="security" src="https://github.com/FuelLabs/fuels-ts/assets/26660/dd2aa6b4-a389-4ea2-8a81-80124639b6f3">


```suggestion
# Security Policy

The Fuel Typescript SDK is still in development, and we take security very seriously.

## Security Issues

If you find a vulnerability or exploit, please report it immediately and privately:
 - https://github.com/FuelLabs/fuels-ts/security/advisories/new

      > Please **DO NOT** file a public issue.

## Non-Security Issues

If the issue is not security-related, please report it publicly:
 - https://github.com/FuelLabs/fuels-ts/issues/new
```